docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -741,17 +782,15 @@ Start with adding the following dependency:
 testImplementation("io.quarkus:quarkus-test-keycloak-server")
 ----
 
-which provides `io.quarkus.test.keycloak.server.KeycloakTestResourceLifecycleManager` - an implementation of `io.quarkus.test.common.QuarkusTestResourceLifecycleManager` which starts a Keycloak container.
```

Comment:
Please keep it but if you'd like, start with `It provides` instead of `which provides`